### PR TITLE
Service: RGB LED

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/led.md [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
-`(0, 10ms), (0.5, 5ms), (0.5, 10ms), (1, 5ms), (0, 7ms), (0, 0ms)`
-will gradually rise the intensity `0 - 0.5` in 10ms,
-then, it will keep it steady for at `0.5` for 5ms,
-then it will rise it again `0.5 - 1` over 10ms,
-drop `1 - 0` in 5ms,
-and keep it at 0 for 7ms more.
-Any entry with duration of 0ms is considered to be end-marker.
-
-To get steady glow at `x`, use animation of `(x, 60000ms), (x, 0ms)` and keep `max_iterations` at `0xffff`.
+Animations are described using pairs of HSV and duration, similarly to the LED service.
```
Comment (mmoskal) [inline services/led.md:10]:
```suggestion
Animations are described using pairs of HSV and duration, similarly to the `status_light` register in the control service.
```

File: services/led.md [hunk 2]
```diff
@@ -51,3 +44,7 @@ If `current_iteration > max_iterations`, then no animation is currently running.
     rw max_iterations = 0xffff: u16 @ 0x81
 
 The animation will be repeated `max_iterations + 1` times.
+
+    const led_count?: u16 @ 0x82
+
+If know, specifies the number of LEDs in parallel on this device.
```
Comment (mmoskal) [inline services/led.md:50]:
```suggestion
If known, specifies the number of LEDs in parallel on this device.
```

File: services/led.md [hunk 3]
```diff
@@ -1,23 +1,13 @@
...
-then, it will keep it steady for at `0.5` for 5ms,
-then it will rise it again `0.5 - 1` over 10ms,
-drop `1 - 0` in 5ms,
-and keep it at 0 for 7ms more.
-Any entry with duration of 0ms is considered to be end-marker.
-
-To get steady glow at `x`, use animation of `(x, 60000ms), (x, 0ms)` and keep `max_iterations` at `0xffff`.
+Animations are described using pairs of HSV and duration, similarly to the LED service.
 
 ## Registers
```
Comment (mmoskal) [inline services/led.md:12]: it could have `max_power` register like the pixel led service

**Comment by pelikhan** [comment]
- use the same format as the status light 
- even for mono-light
- enum led type
- enum for shape?
- number of leds
-

**Comment by pelikhan** [comment]
@mmoskal i've added engouh const info in the register to correctly handle monochrome/rgb LEDs in the UI. So we can have a single service to drive LED/strips.

File: services/led.md (hunk 3 above)
Comment (pelikhan) [reply services/led.md:8]: yup it's there.

**Pull request merged**